### DISCUSSION TITLE
Show active frequency for visitors?

### DISCUSSION THREAD

**Discussion by HB9ERD** [magicbug/Cloudlog#996] [Ideas / Feature Requests]
Hi, I am a new Cloudlog user

I think it would be nice if visitors of my cloudlog page could see my active frequency (to contact me on the bands) when I am active. Is that possible? Via CloudlogCAT and the API, the website knows on which QRG an mode I am currently operating. So, it could show something like "currently active on 14.235 MHz in SSB". Of course, if there is nothing coming in via the API, it could also show "not active".

73
Udo, HB9ERD

**Comment by magicbug** [comment]
Udo,

Currently its not possible, but I see no reason not to add it.

Peter.

**Comment by magicbug** [comment]
I've added this to the 2021 roadmap https://github.com/magicbug/Cloudlog/projects/8

**Comment by HB9ERD** [comment]
Thanks for adding it to the roadmap with medium priority. That will be nice when it is available and I think no other logbook has such a unique feature. Will be a great "plus".

**Comment by magicbug** [comment]
OK got a bit bored, Cloudlog now shows "Radio Status" on the dashboard for radio interfaces that have been active in the last 15mins.

![Screenshot from 2021-05-03 14-03-04](https://user-images.githubusercontent.com/84308/116879971-4b27ee00-ac19-11eb-975c-37a8ea8d548d.png)

**Comment by HB9ERD** [comment]
Very nice that you could add this so quickly. Many thanks! It works very well.
